pkg/chartutil/values.go

```diff
@@ -350,13 +365,14 @@ type ReleaseOptions struct {
 // In Helm 3.0, this will be changed to accept Capabilities as a fourth parameter.
 func ToRenderValues(chrt *chart.Chart, chrtVals *chart.Config, options ReleaseOptions) (Values, error) {
 	caps := &Capabilities{APIVersions: DefaultVersionSet}
-	return ToRenderValuesCaps(chrt, chrtVals, options, caps)
+	secrets := &chart.Config{Type: chart.ConfType_SECRETS}
+	return ToRenderValuesCaps(chrt, chrtVals, secrets, options, caps)
 }
 
 // ToRenderValuesCaps composes the struct from the data coming from the Releases, Charts and Values files
 //
 // This takes both ReleaseOptions and Capabilities to merge into the render values.
-func ToRenderValuesCaps(chrt *chart.Chart, chrtVals *chart.Config, options ReleaseOptions, caps *Capabilities) (Values, error) {
+func ToRenderValuesCaps(chrt *chart.Chart, chrtVals *chart.Config, chrtSecrets *chart.Config, options ReleaseOptions, caps *Capabilities) (Values, error) {
```

Comment:
This is a breaking public API change. This doesn't mean we can't pull it in to 2.X, but we have to have a consensus among all of us that this is ok.